openapi/api/openapi.yaml

```diff
@@ -14,7 +14,7 @@ info:
   title: REST API
   version: v1
 servers:
-- url: https://api.rs2.usw2.rockset.com/
+- url: https://api.use1a1.rockset.com/
```

Comment:
Is this right? Why not usw2a1?

Reply:
I've got a linear issue to fix that, this diff is just an openapi update